hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -815,5 +815,57 @@
       stage: V2
       tests: ["usbdev_nak_to_out_trans_when_avbuffer_empty_rxfifo_full"]
     }
+    {
+      name: data_toggle_restore
+      desc: '''
+            Verify that each of the IN and OUT Data Toggle bits may be both set and cleared by
+            software to ensure that resuming communications after returning from Deep Sleep
+            is possible.
+
+            - Ensure that the USB device disconnected (pull up not enabled) such that no
+              communications shall occur with the device.
+            - Set all of the even-numbered OUT Data Toggle bits to 1 and all of the odd-numbered
+              OUT Data Toggle bits to 0.
+            - Check that the OUT Data Toggle bits read as 0x555.
+            - Set all of the even-numbered IN Data Toggle bits to 0 and all of the odd-numbered
+              IN Data Toggle bits to 1.
+            - Check that the IN Data Toggle bits read as 0xAAA.
+            - Check that the OUT Data Toggle bits read as 0x555.
+            - Invert all of the OUT Data Toggle bits.
+            - Invert all of the IN Data Toggle bits.
+            - Check that the OUT Data Toggle bits read as 0xAAA.
+            - Check that the IN Data Toggle bits read as 0x555.
+            - Set all of the OUT Data Toggle bits to random values.
+            - Set all of the IN Data Toggle bits to random values.
+            - Check that all of the OUT and IN Data Toggle bits read back as expected.
+            '''
+      stage: V2
+      tests: ["usbdev_data_toggle_restore"]
```

Comment:
I don't think that test exists yet does it? This field is supposed to be "here is a list of tests that implement the testpoint". If we haven't written one yet, I think the list should be empty. Similar for `setup_priority` below.

Reply:
Right. I've removed the test names for the 3 tests I've altered, but not updated the other entries which I had mimicked.